spec.html

```diff
@@ -26024,6 +26140,17 @@ <h1>Abstract Module Records</h1>
                 Purpose
               </th>
             </tr>
+            <tr>
+              <td>
+                GetModuleSource()
+              </td>
+              <td>
+                <p>It returns either a normal completion completion for the ModuleSource Object corresponding to this source Module Record's source phase (<emu-xref href="#sec-module-source-objects"></emu-xref>), or a throw completion.</p>
```

Comment:
```suggestion
                <p>It returns either a normal completion containing the Module Source Object corresponding to this Module Record, or a throw completion.</p>
```

Assuming we `<dfn>` "Module Source Object" below.